package.json

```diff
@@ -51,6 +51,10 @@
     "react-styleguidist": "11.0.5",
     "source-map-explorer": "2.4.2"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/IagoRegiani/verificador-associacao.git",
+  },
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/manual/iagoregiani/verificador-associacao/pullRequest?prid=5514387)